docker-compose.yml

```diff
@@ -36,6 +36,8 @@ services:
       - POSTGRES_HOST=db
     depends_on:
       - db
+    networks:
```

Comment:
Do we need to add a network? Containers should be connected by default I'd be interested to see if it works without a network

Reply:
I added it because I initially had some trouble connecting to the db. I don't think removing this will cause any issues and since I can't get docker to work post folder shift, I might as well remove it. 🤷